plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -342,7 +342,7 @@ func ClusterRoles() []rbac.ClusterRole {
 				// this is for leaderlease access
 				// TODO: scope this to the kube-system namespace
 				rbac.NewRule("create").Groups(legacyGroup).Resources("endpoints").RuleOrDie(),
-				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
+				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-default-scheduler").RuleOrDie(),
```

Comment:
kube-scheduler's not namespace level, multi-scheduler also uses `schedulerName` instead of namespace; so we're using different name :). But unfortunately, there's not feature to guarantee unique name (e.g. admission).